src/agent/src/InputProcessor/SystemPromptInputProcessor.php

```diff
@@ -45,7 +49,9 @@ public function processInput(Input $input): void
             return;
         }
 
```

Comment:
lets add a validation and throw if translator is null and enable translation is true, WDYT?